__tests__/api.test.js

```diff
@@ -196,13 +196,14 @@ describe("GET /api/articles/:article_id/comments", () => {
     })
 })
 // Ticket7
-describe.skip("POST /api/articles/:article_id/comments", () => {
+describe("POST /api/articles/:article_id/comments", () => {
     it("Should post a simple comment to an article that exists", () => {
         return request(app)
         .post("/api/articles/2/comments")
         .send({user_name: "butter_bridge", body: "TEST POST PLEASE IGNORE"})
         .expect(201)
         .then(({body}) => {
+            console.log(body)
             expect(body).toHaveProperty("article_id")
             expect(body).toHaveProperty("author")
             expect(body).toHaveProperty("body")
```

Comment:
I'd add another 201 test here that ignores additional properties such as:
**{
        username: "bainesface",
        body: "I loved this game too!",
        extraProperty: "extra things",
      }**
      
Please add a couple of tests for:

Status 400, missing required field(s), e.g. no username or body properties
Status 404, username does not exist